clmm/theory/ccl.py

```diff
@@ -107,6 +115,8 @@ def _set_concentration(self, cdelta):
         """set concentration. Also sets/updates hdpm"""
         # pylint: disable=protected-access
         self.conc = ccl.halos.ConcentrationConstant(c=cdelta, mdef=self.mdef)
+        if parse(ccl.__version__) > parse("2.7.0"):
+            ccl.UnlockInstance.Funlock(type(self.mdef), "_concentration_init", True)
```

Comment:
@m-aguena I am thinking if I should remove those two lines and make CLMM only support CCL 2.6 to 2.7, not the dev versions.

Reply:
Just leave it there for now. It will be removed when we update to CCL 3.0, right?

Reply:
For CCL v3, line 119 and 120 will become
```
with ccl.UnlockInstance(self.mdef):
      self.mdef.concentration = self.conc
```
and the if statement in line 118 will be removed